server/vulnerabilities/cpe.go

```diff
@@ -139,67 +114,328 @@ func cleanAppName(appName string) string {
 	return strings.TrimSuffix(appName, ".app")
 }
 
-var onlyAlphaNumeric = regexp.MustCompile("[^a-zA-Z0-9]+")
-
-func CPEFromSoftware(db *sqlx.DB, software *fleet.Software) (string, error) {
-	targetSW := ""
-	switch software.Source {
-	case "apps":
-		targetSW = "macos"
-	case "python_packages":
-		targetSW = "python"
-	case "chrome_extensions":
-		targetSW = "chrome"
-	case "firefox_addons":
-		targetSW = "firefox"
-	case "safari_extensions":
-		targetSW = "safari"
-	case "deb_packages":
-	case "portage_packages":
-	case "rpm_packages":
-	case "npm_packages":
-		targetSW = `"node.js"`
-	case "atom_packages":
-	case "programs":
-		targetSW = `"windows*"`
-	case "ie_extensions":
-	case "chocolatey_packages":
+var onlyAlphaNumeric = regexp.MustCompile(`[^a-zA-Z0-9]+`)
```

Comment:
Nit: The var name is confusing :) (this is actually `nonAlphaNumeric`)

Reply:
Fixed